webserver/tests/http2/src/test/java/io/helidon/webserver/tests/http2/HeadersTest.java

```diff
@@ -76,25 +87,67 @@ static void setUpServer(WebServerConfig.Builder serverBuilder) {
 
     @SetUpRoute
     static void router(HttpRouting.Builder router) {
+        router.error(IllegalStateException.class, (req, res, t) -> res.status(500).send(t.getMessage()));
         router.route(Http2Route.route(GET, "/ping", (req, res) -> res.send("pong")));
         router.route(Http2Route.route(GET, "/cont-out",
-                (req, res) -> {
-                    for (int i = 0; i < 500; i++) {
-                        res.header("test-header-" + i, DATA + i);
-                    }
-                    res.send();
-                }
+                                      (req, res) -> {
```

Comment:
This reformatting is not needed, as there is no changed code here, and it is probably different than the formatting in the project.